### ISSUE TITLE
More arrow ending shapes 

### ISSUE THREAD

**Issue by mgudmund** [excalidraw/excalidraw#3167]
I would like to see a few more arrow-ending options be added. 
I would like to see some more, such as hollow circle, and half-circle. 

Is this something that could be added?

Is there an existing plan to add more, or should I try to provide som commonly used in my work as an enterprise architect?

**Comment by mgudmund** [comment]
Example of a common use case when discussing software architecture: 
![image](https://user-images.githubusercontent.com/3577698/109808394-58c7f400-7c27-11eb-8499-cd77b7096401.png)

**Comment by baldurh** [comment]
I’d love to revive this discussion. Especially I’m interested in the crow's foot notation as suggested in https://github.com/excalidraw/excalidraw/issues/616

I’m up for contributing on this but it seems all the issues and PRs strand on the interface part. Can we come to a conclusion on that so this can be taken forward?

This is how it looks currently
<img width="195" alt="image" src="https://user-images.githubusercontent.com/1823617/161514331-241ddba8-d34b-4105-9f14-bb8c4bde888f.png">

Do we want to stick with the 3x3 grid or change it to 4x4? Do we want to introduce some arrowhead "sets" so that you can switch between sets (standard, crow's foot, etc.).

Something like
<img width="248" alt="image" src="https://user-images.githubusercontent.com/1823617/161516660-9187aa0e-5e7a-4eb4-ad14-e470e8bb6866.png">

or using an icon

<img width="247" alt="image" src="https://user-images.githubusercontent.com/1823617/161517387-edc29206-a883-4277-949b-2dbc91c0b063.png">

(disclaimer: obviously not a UX designer 🙈)

That way we could eventually support all kinds of notations eg. different entity-relationship notations:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/1823617/161514913-125223f3-2476-4d9a-beab-1f575617e5ba.png">

What do you think @dwelle @lipis @steveruizok ?

Links to other issues and PRs related to this:
https://github.com/excalidraw/excalidraw/issues/616
https://github.com/excalidraw/excalidraw/issues/839
https://github.com/excalidraw/excalidraw/pull/2486
https://github.com/excalidraw/excalidraw/pull/3477

**Comment by cocoonkid** [comment]
Just today we discussed replacing drawio with excalidraw but the lack of various arrowenotations so far isunfortunately  a stopper rn.


![image](https://user-images.githubusercontent.com/43231640/221795072-70043fad-8e38-4553-b24e-e992ba728151.png)


looking forward to see these implemented :)

**Comment by peterhungnxp** [comment]
Please add this, super useful!!!
![image](https://user-images.githubusercontent.com/72534365/228121010-0a41f5ae-f26f-4e1d-b70d-eb34684b5614.png)

**Comment by edmondchuc** [comment]
Please also consider [UML class diagram arrowhead relationships](https://en.wikipedia.org/wiki/Class_diagram#Relationships).

![Uml_classes_en](https://github.com/excalidraw/excalidraw/assets/37032744/68e355fb-0392-488c-abc0-b9b10c69fcf3)

**Comment by vizvamitra** [comment]
I also miss crowfoot notation a lot, it is much simpler than adding all those "1"s and "n"s. Also might be useful to add hotkeys for start/end arrowheads

**Comment by AndrewBasem1** [comment]
Still a much needed feature! please add crowfoot notation. (also thank you for all the hardwork)

**Comment by georgekenessey** [comment]
Yes this would be great, we would love to replace Lucidchart fully but the remaining thing we need to is crowfoot notation so we can create entity diagrams.

**Comment by dwelle** [comment]
We've recently added more arrowheads so closing for now (https://x.com/excalidraw/status/1876276078177816692)

**Comment by georgekenessey** [comment]
@dwelle thank you so much! We have seen it show up and have already been using it. You guys are the best. It is so appreciated!!!

**Comment by botagas** [comment]
I'd like to revive the discussion. Many arrowheads are still missing from Crow's Foot notation. There is no way to signify 0 to many relationship at the moment, only one to many.
